config/checkstyle_checks.xml

```diff
@@ -155,6 +141,20 @@
              value="The warning ''{0}'' cannot be suppressed at this location. Only few javac warnings are allowed to suppress. If try to suppress checkstyle/pmd/..... violation please do this in their config file. If you try to suppress IntelliJ IDEA inspection, please use javadoc block tag @noinspection"
              />
     </module>
+    <module name="SuppressionCommentFilter">
```

Comment:
These are filters not annotations as the header on line 122 suggest.
Make a new header for filters inside `TreeWalker` and move these lines right under it.